README.md

```diff
@@ -1,46 +1,429 @@
+
+One thing to remember:  All token and reputation amounts should be expressed in Wei.
+
+## Working with DAOs
+
+Arc.js provides a class named "DAO" that enables you to create new DAOs and obtain information about them.
+
+### Create a new DAO with all defaults
+
+The simplest example of how to create a new DAO uses all defaults:  no schemes nor founders, using the Universal Controller and default DaoCreator scheme.  See `NewDaoConfig`.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT"
+});
+```
+
+Add and remove schemes later using `SchemeRegistrar`.
+
+### Create a new DAO with founders
+
+Create a new DAO with founders by including a "founders" array.  This will automatically mint tokens and reputation to each founder.   See `NewDaoConfig`.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  founders: [
+    {
+      // the current user
+      address: accounts[0],
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: anyAddress,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: anyAddress,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    }
+  ]
+});
+```
+
+### Create a new DAO with schemes
+
+Create a new DAO with schemes by including a "schemes" array.  This will register the schemes with all default permissions and voting machines. See `SchemesConfig` and `SchemeConfig`.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    { name: "GlobalConstraintRegistrar" }
+  ]
+});
+```
+
+### Create a new DAO overriding the default voting machine
+
+Override the default voting machine by adding a "votingMachines" element.  This will override the default voting machine, either at the root level to apply to all schemes, or within each scheme element to override for the specific scheme. See `NewDaoVotingMachineConfig` and `SchemesConfig`.
+
+#### Default root-level override for all schemes
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
+    ownerVote:false
+  }
+});
 ```
 
-You can also override the default configuration settings with environment variables:
+#### Scheme-specific override
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    {
+      name: "GlobalConstraintRegistrar",
+      votingMachineParams: {
```

Comment:
how do you change the votingMachine scheme used?

Reply:
`NewDaoVotingMachineConfig.votingMachineAddress`.  Are you suggesting this should be documented?

Reply:
Done